config/config.go

```diff
@@ -341,6 +366,20 @@ func (c *Config) CheckWebserverConfigValues() error {
 	return nil
 }
 
+// CheckProviderConfigValues checks Provider configuration values
+func (c *Config) CheckProviderConfigValues() error {
```

Comment:
A check here for default values would be good. Fatal error if the default values are set without the users ones unless the forex provider allows forex conversions without an API key.